.github/workflows/ci.yml

```diff
@@ -35,6 +41,9 @@ jobs:
       - name: Check Formatting
         run: terraform fmt -recursive -check -no-color
 
+      - name: List all Stacks
+        run: terramate list
+
```

Comment:
should we also add terramate fmt and generation checks?

Reply:
We already have fmt checking:https://github.com/mineiros-io/terramate-example-code-generation/pull/18/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR42

Adding generation check would be a good idea, we don't have a check yet, like fails if there is pending code to generate, but we would run generate for now to be sure it is not broken at least.

Reply:
Its true :laughing: . Will add one for Terramate fmt =P